check_dependencies.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if ! command -v pciutils i2c-tools mesa-utils smartmontools dmidecode >/dev/null 2>&1; then
```

Comment:
Does this work if a subset of these packages is installed?

Reply:
From `man command` it appears these should be command names (not package names) and only one at a time can be tested: `command [−p][−v|−V] command_name`

Reply:
We have checked and the current command effectively checks for only the first dependency (`pciutils`). This obviously has to be fixed before merging.